File: Bridge/Resources/.generated/system/threading/synchronizationContext.js
    Bridge.define("System.Threading.SynchronizationContext", {
        statics: {
            props: {
                Current: {
                    get: function () {
                        throw new System.NotImplementedException();
                    }
                }
            },
            methods: {
                setSynchronizationContext: function (syncContext) {
                    throw new System.NotImplementedException();
                },
                waitHelper: function (waitHandles, waitAll, millisecondsTimeout) {
                    throw new System.NotImplementedException();
                }
            }
        },
        ctors: {
            ctor: function () {
                this.$initialize();
            },
            $ctor1: function (context) {
                this.$initialize();
            }
        },
        methods: {
            createCopy: function () {
                throw new System.NotImplementedException();
            },
            isWaitNotificationRequired: function () {
                throw new System.NotImplementedException();
            },
            operationCompleted: function () {
                throw new System.NotImplementedException();
            },
            operationStarted: function () {
                throw new System.NotImplementedException();
            },
            post: function (d, state) {
                throw new System.NotImplementedException();
            },
            send: function (d, state) {
                throw new System.NotImplementedException();
            },
            setWaitNotificationRequired: function () {
                throw new System.NotImplementedException();
            },
            wait: function (waitHandles, waitAll, millisecondsTimeout) {
                throw new System.NotImplementedException();
            }
        }
    });

File: Bridge/System/Threading/SynchronizationContext.cs
using Bridge;

namespace System.Threading
{
    public class SynchronizationContext
    {
        public SynchronizationContext()
        {
        }

        internal SynchronizationContext(SynchronizationContext context)
        {
        }

        public static SynchronizationContext Current
        {
            get
            {
                throw new NotImplementedException();
            }
        }

        public virtual SynchronizationContext CreateCopy()
        {
            throw new NotImplementedException();
        }

        public bool IsWaitNotificationRequired()
        {
            throw new NotImplementedException();
        }

        public virtual void OperationCompleted()
        {
            throw new NotImplementedException();
        }

        public virtual void OperationStarted()
        {
            throw new NotImplementedException();
        }

        public virtual void Post(SendOrPostCallback d, object state)
        {
            throw new NotImplementedException();
        }

        public virtual void Send(SendOrPostCallback d, object state)
        {
            throw new NotImplementedException();
        }

        public static void SetSynchronizationContext(SynchronizationContext syncContext)
        {
            throw new NotImplementedException();
        }

        protected void SetWaitNotificationRequired()
        {
            throw new NotImplementedException();
        }

        public virtual int Wait(IntPtr[] waitHandles, bool waitAll, int millisecondsTimeout)
        {
            throw new NotImplementedException();
        }

        protected static int WaitHelper(IntPtr[] waitHandles, bool waitAll, int millisecondsTimeout)
        {
            throw new NotImplementedException();
        }
    }
}